packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx

```diff
@@ -118,14 +118,14 @@ export const SPAPropertyDetailPage = (): JSX.Element => {
           >
             <Split hasGutter className="pf-u-mt-md">
               <SplitItem isFilled>
-                <Card style={{ maxWidth: '100%' }} isFullHeight>
+                <Card isFullHeight style={{ height: '320px' }}>
```

Comment:
@akhilmhdh I think we can also try styled components

Reply:
@hybrdix We actually dont need styled component. Its used when u need to style a lot of things. For minor tweeks like custom height just use style props.

1. If its used everywhere like removing anchor text decoration write a utility css class in global file

2. Override patternfly css variable if its used everywhere. As we are using patternfly this is recommended most of the cases.

3. Use nextjs style syntax. (Least needed)

As we are using patternfly and some utility class of pfe introducing styled components is unnecessary bundle.